### PR TITLE
junyu's info updated on home page

### PR DESCRIPTION
**Pull request by DrakeW**

### CODE REVIEW

File: app/views/pages/home.html.slim [hunk 1]
```diff
@@ -96,6 +96,15 @@
...
                     h3 Sam Lau
                     = render partial: "shared/tas/sam"
+            .ta
+              .row
+                .col-md-5
+                  .image-container.image-top.left-side
+                    = image_tag "junyu_wang.jpeg", class: "round-with-border"
+                .col-md-5
+                  .description.right.right-side
+                    h3 Sam Lau
```
Comment (negativetwelve) [inline app/views/pages/home.html.slim]: Make sure this is your name!

**Comment by negativetwelve** [comment]
Can you add *.swp to the gitignore file and then `git rm --cached` the .swp files?

**Comment by DrakeW** [comment]
Sure I'll do it tmr morning

**Comment by DrakeW** [comment]
Ops and I forgot to change the name...I'll fix them all tmr thanks for telling me!

**Comment by negativetwelve** [comment]
:+1:

**Comment by DrakeW** [comment]
Hey Mark I've updated the pull request, all .swp are removed and name has been fixed

**Comment by negativetwelve** [comment]
Looks good to me, I'll let @chenhoward merge and push this to production.

**Comment by chenhoward** [comment]
Sorry I didn't get to look until now, but on my browser it appears as an oval rather than a circle.  Also we should continue the left right offsets of TA's.

**Comment by DrakeW** [comment]
I just fixed the left right offsets, and shrink the image size, maybe this time it will be in a circle

**Comment by negativetwelve** [comment]
Have you looked at it locally? If not, you should set up the app on your computer (plus it'll help when you need to debug students getting rails up on their machines)

**Comment by DrakeW** [comment]
Hi I've tested the app on my laptop and now everything (left-right-offset and circle image) has been fixed

**Pull request merged**